tests/capture/test_capture_cond.py

```diff
@@ -645,6 +645,7 @@ def test_circuit_consts(self, pred, arg, expected):
         res_ev_jxpr = jax.core.eval_jaxpr(jaxpr.jaxpr, jaxpr.consts, *args)
         assert np.allclose(res_ev_jxpr, expected), f"Expected {expected}, but got {res_ev_jxpr}"
 
+    @pytest.mark.xfail(strict=False)  # might pass if postselection equal to measurement
```

Comment:
I'm not a big fan of selectively setting `strict=False`. It will make it harder for us to come back and un-xfail this test in the future. Maybe we can leave this how you've done it for now and come back in a later PR to split the test into two where one should always pass and the other should always fail.

If you like that idea, could you create a backlog story for this?

Reply:
I think when we add mcm support, we should just search for all the `xfail` occurances.  While there's several of them, they are relatively localized to mcm tests in the catpure module.